src/main/resources/add-ons-to-index.json

```diff
@@ -2279,6 +2302,7 @@
       "tags": [
         "data-management"
       ],
+      "icon": "puzzle-piece",
```

Comment:
Good choice but I feel "link" is a better option as it represents the utility of the module better. Kindly check https://wiki.openmrs.org/display/docs/Patient+Matching+Module for more insight